lua/entities/marums_hoverball/cl_init.lua

```diff
@@ -3,4 +3,5 @@ include('shared.lua')
 function ENT:Draw()
     -- self.BaseClass.Draw(self) -- Overrides Draw
     self:DrawModel() -- Draws Model Client Side
+    if WireLib then Wire_Render(self.Entity) end
```

Comment:
could you give a quick explanation of what wire_render does?

Reply:
Since its not in any of the default wire entities, ill remove this.
I was taking an example from pewpew. I mightve wanted to do a little more research into the reasons.